src/TesterInternal/RemindersTest/ReminderTablePluginTests.cs

```diff
@@ -9,32 +9,82 @@ namespace UnitTests.RemindersTest
 {
     internal class ReminderTablePluginTests
     {
-        internal static async Task ReminderTableUpsertParallel(IReminderTable reminder)
+        internal static async Task ReminderTableTest(IReminderTable reminderTable)
         {
-            var grainRef = MakeTestGrainReference();
-            var period = TimeSpan.FromMinutes(1);
-            var reminderName = "testReminderName";
-            var startAt = DateTime.UtcNow.Add(TimeSpan.FromMinutes(1));
-            var results = await Task.WhenAll(Enumerable.Range(0, 10).Select(x => UpsertReminder(reminder, grainRef, reminderName, startAt, period)));
+            //int.MinValue == -2^31 == -2147483648
 
-            Assert.IsTrue(results.Distinct().Count() == results.Length);
+            Guid guid0 = new Guid("00000000-0000-0000-0000-000000000001");
```

Comment:
Lets also have one case with Random Guid, like before, in addition to cases you added.